associacao-brasileira-de-normas-tecnicas-ufg-fct.csl

```diff
@@ -0,0 +1,603 @@
+              <text macro="access"/>
+            </group>
+          </group>
+        </else-if>
+        <else-if type="article-magazine article-journal" match="any">
+          <group suffix=".">
+            <text macro="author" suffix=". "/>
+            <text macro="title" suffix=", "/>
+            <text macro="container-title" suffix=", "/>
+            <text variable="collection-title" suffix=". "/>
+            <!-- use field 'extra' place -->
+            <text variable="note" suffix=", "/>
+            <text macro="edition" suffix=", "/>
+            <text macro="locators" suffix=", "/>
+            <text macro="issued" suffix=". "/>
+            <text variable="DOI" prefix="DOI: " suffix=""/>
+            <text macro="access"/>
+          </group>
+        </else-if>
+        <else-if type="thesis">
+          <group>
+            <text macro="author" suffix=". "/>
+            <text macro="title" suffix=". "/>
+            <text macro="issued-year" suffix=". "/>
+            <text variable="number-of-pages" suffix=" f. "/>
+            <text variable="genre" suffix=" &#8211; "/>
+            <text variable="publisher" suffix=", "/>
+            <text variable="publisher-place" suffix=", "/>
+            <text macro="issued-year" suffix=". "/>
+            <text variable="DOI" prefix="DOI: " suffix=". "/>
+            <text macro="access" suffix=". "/>
+          </group>
+        </else-if>
+        <else-if type="manuscript">
+          <!-- There is no ABNT style for manuscripts.-->
+          <group>
+            <text macro="author" suffix=". "/>
+            <text macro="title" suffix=". "/>
+            <text macro="edition" suffix=". "/>
+            <text macro="abstract" suffix=". "/>
+            <text macro="place" suffix=", "/>
+            <text macro="issued" suffix=". "/>
+            <text macro="access" suffix=". "/>
+            <text macro="archive" suffix=". "/>
+          </group>
+        </else-if>
+        <else-if type="webpage">
+          <group>
+            <text macro="author" suffix=". "/>
+            <text macro="title" suffix=". "/>
+            <text macro="issued" suffix=". "/>
+            <text macro="container-title" suffix=". "/>
+            <text macro="genre" prefix=" [" suffix="]. "/>
+            <text variable="DOI" prefix="DOI: " suffix=". "/>
+            <text macro="access" suffix="."/>
+            <!-- On Zotero, use field 'extra' for notes.-->
+            <text variable="note" prefix=" (" suffix=")."/>
+          </group>
+        </else-if>
+        <else-if type="paper-conference">
+          <text macro="author" suffix=". "/>
+          <text macro="title" suffix=". "/>
+          <text macro="event" suffix=", "/>
+          <text variable="volume" suffix=", "/>
+          <text macro="issued" suffix=". "/>
+          <!-- On Zotero, use field 'extra' for notes/event location.-->
+          <text variable="note" suffix=". "/>
```

Comment:
Can you clarify what this is for and give some examples?

Reply:
The lines +538 to +539 in the CSL file provide instructions for using the 'extra' field in Zotero to capture notes or event location information. Specifically, these lines indicate that any notes or event location details entered in the 'extra' field of Zotero will be included in the citation, followed by a period.

Explanation and Examples:
Purpose:

These lines ensure that any additional information such as notes or event locations entered in the 'extra' field in Zotero is incorporated into the citation. This is useful for providing extra context or details that are not typically included in standard citation fields.
Example Usage:

Suppose you are citing a conference presentation and want to include the event location. You can enter this information in the 'extra' field in Zotero, and it will appear in the citation.
Practical Example:

In Zotero:
Title: "Advances in AI"
Author: "John Doe"
Year: 2024
Extra: "Presented at the International Conference on Artificial Intelligence, San Francisco, CA."
Generated Citation:
Doe, John. "Advances in AI." Presented at the International Conference on Artificial Intelligence, San Francisco, CA. 2024.
By using the 'extra' field, users can add specific notes or event locations which are then seamlessly integrated into the citation, enhancing the information provided to the reader without cluttering the primary citation fields.